contrib/cpp/src/python/pants/contrib/cpp/rules/targets.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    Dependencies,
+    Sources,
+    StringSequenceField,
+    Target,
+)
+
+
+class CppLibraries(StringSequenceField):
```

Comment:
Nit: `CppLibraries` makes me think of "a collection of `CppLibrary`, which is not what this is. Could we rename it to `ExternalCppLibraries`?

Reply:
Sure, great idea!